src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/diagnostics/rules/MissingJavadocCodeSnippetsRule.java

```diff
@@ -18,6 +17,9 @@
 public class MissingJavadocCodeSnippetsRule implements DiagnosticRule {
     public static final String CODE_SNIPPET_TAG = "{@codesnippet";
 
+    private static final Pattern NEW_CODESNIPPET_TAG = Pattern
+        .compile("(\\s*)\\*?\\s*<!--\\s+src_embed\\s+([a-zA-Z0-9.#\\-_]+)\\s*-->");
```

Comment:
is there a way to add some unit tests for this pattern?

Reply:
I just copied this out of the codesnippet-maven-plugin, so it should work on matching that. But there is a question on how to make sure these stay consistent with each other.